### ISSUE TITLE
startup: replace the freezing status splash logo with a generic (rotating) Animator

### ISSUE THREAD

**Issue by caybro** [status-im/status-desktop#9524]
Rationale: we don't want to display a half animated image that freezes in the middle of the loading process due to NIM backend blocking the main thread

**Comment by caybro** [comment]
Could look something like this (the thing is spinning the whole time until the app is loaded :)

![image](https://user-images.githubusercontent.com/5377645/218129118-bf9c19bf-0097-49e9-96cb-af5c31f1ca50.png)

CC @benjthayer

**Comment by caybro** [comment]
Or even better, like this ;)

![image](https://user-images.githubusercontent.com/5377645/218141340-2bbde054-75ac-4d3e-a009-e862b0bb2503.png)

**Comment by John-44** [comment]
Agreed 2nd is better 🙂   

@caybro  are there any constraints on the size of the Status logo with this technique e.g. would it be possible to make the size of the spinning status logo larger so that it is sized similarly to the previous designs?

@alexandraB99 @alexjba  the size of the Status logo on the proceeding login screen should be (if possible) the same size as the spinning Status logo on @caybro's spinning status logo shown above (which hopefully @caybro can make larger).  Even better would be if we use the actual exact same rendering (that @caybro is using on the 'loading status' screen) on the preceding login screen (but not spinning) so that when the user transitions from the login screen to the 'loading status' screen, the user will only see the Status logo start spinning, and no other changes.

And then after the initial loading is complete and we transition to the 'fake loading' screen with the messages ( https://github.com/status-im/status-desktop/issues/9470 ), this fake loading screen should also use the exact same gpu spinner that @caybro has created here.

So to summarize:

Login screen (should use the @caybro's status logo gpu spinner, but not spinning) --> Loading screen (w/ status logo gpu spinner spinning) --> Fake loading screen with messages  (should also use @caybro's status logo gpu spinner spinning)

**Comment by caybro** [comment]
> Agreed 2nd is better slightly_smiling_face
> 
> @caybro are there any constraints on the size of the Status logo with this technique e.g. would it be possible to make the size of the spinning status logo larger so that it is sized similarly to the previous designs?
> 
> @alexandraB99 @alexjba the size of the Status logo on the proceeding login screen should be (if possible) the same size as the spinning Status logo on @caybro's spinning status logo shown above (which hopefully @caybro can make larger). Even better would be if we use the actual exact same rendering (that @caybro is using on the 'loading status' screen) on the preceding login screen (but not spinning) so that when the user transitions from the login screen to the 'loading status' screen, the user will only see the Status logo start spinning, and no other changes.
> 
> And then after the initial loading is complete and we transition to the 'fake loading' screen with the messages ( #9470 ), this fake loading screen should also use the exact same gpu spinner that @caybro has created here.
> 
> So to summarize:
> 
> Login screen (should use the @caybro's status logo gpu spinner, but not spinning) --> Loading screen (w/ status logo gpu spinner spinning) --> Fake loading screen with messages (should also use @caybro's status logo gpu spinner spinning)

TLDR, yeah we can make the logo any size and use it also for the subsequent fake loading screen

**Comment by caybro** [comment]
@John-44 just need to know what size it should be :)

**Comment by caybro** [comment]
According to https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=1267%3A320303&t=gOEZ1ivQcx4PTsb6-0, it should be 128px

**Comment by John-44** [comment]
> @John-44 just need to know what size it should be :)

Ideally the same size as shown in this figma https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=1267%3A320303&t=eHRxT6iQxaouefws-1

Figma says it's 128px X 128px, at standard screen density I assume.

**Comment by caybro** [comment]
> > @John-44 just need to know what size it should be :)
> 
> Ideally the same size as shown in this figma https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=1267%3A320303&t=eHRxT6iQxaouefws-1
> 
> Figma says it's 128px X 128px, at standard screen density I assume.

Code in master says it's also 128x128 ;) 

https://github.com/status-im/status-desktop/blob/master/ui/app/mainui/SplashScreen.qml#L14

**Comment by John-44** [comment]
> > > @John-44 just need to know what size it should be :)
> > 
> > 
> > Ideally the same size as shown in this figma https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=1267%3A320303&t=eHRxT6iQxaouefws-1
> > Figma says it's 128px X 128px, at standard screen density I assume.
> 
> Code in master says it's also 128x128 ;)
> 
> https://github.com/status-im/status-desktop/blob/master/ui/app/mainui/SplashScreen.qml#L14

Nice :-)
